src/main/docker/Dockerfile

```diff
@@ -8,7 +10,7 @@ ENV SPRING_OUTPUT_ANSI_ENABLED=ALWAYS \
 ENV spring.profiles.active ""
 
 RUN echo "Installing needed dependencies" \
-  && apt-get update && apt-get install -y --no-install-recommends locales graphviz wget \
+  && apt-get update && apt-get install -y --no-install-recommends locales graphviz wget unzip \
```

Comment:
Mhh not sure if we can get rid of the unzip as it's not necessary for production systems of Artemis? At least it has to be documented what this is used for.

Reply:
@4ludwig4 I have removed any changes to the Dockerfile